ui/public/index.html

```diff
@@ -2,7 +2,27 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link name="unprocessed-script" content="lockdown.umd.js" />
+    <script src="./lockdown.umd.js"></script>
+    <script>
+      // Allow the React dev environment to extend the console for debugging
+      // features.
+      const consoleTaming = '%NODE_ENV%' === 'production' ? 'safe' : 'unsafe';
```

Comment:
Is `%NODE_ENV%` expansion a feature of the build system that transforms `index.html`?

Reply:
Yep, I think it's from `react-scripts`